### PR TITLE
Update community.eml

### PR DESCRIPTION
**Pull request by AndroGenius-codes**
fix to the responsiveness issues on small screen in community page. @sabine this addresses the community page part of issue #1742

### CODE REVIEW

File: src/ocamlorg_frontend/pages/community.eml [hunk 1]
```diff
@@ -187,8 +187,8 @@ Layout.render
       <h3 class="font-bold mb-6">OCaml Outreachy Internships</h3>
       <div class="text-lg mb-8 lg:px-28">This is a record of all past OCaml Community Outreachy Internship Projects.</div>
       <div class="space-x-0 md:space-x-5 space-y-5 md:space-y-0">
-        <a class="btn btn-lg btn-secondary" href="<%s Url.outreachy %>">
-          <span>View OCaml Outreachy Internship Projects</span>
+        <a class="btn btn-lg btn-secondary px-4" href="<%s Url.outreachy %>">
+          <span class="text-sm">View OCaml Outreachy Internship Projects</span>
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/community.eml]:
```suggestion
        <a class="btn btn-lg btn-secondary" href="<%s Url.outreachy %>">
          <span>View Outreachy Projects</span>
```

**Review by sabine — COMMENTED** [review]
Thanks for identifying the place where the page overflows.

Rather than changing the styles here, I would shorten the button text.

**Review by sabine — APPROVED** [review]
Thanks for the contribution @AndroGenius-codes!

**Comment by AndroGenius-codes** [comment]
> Thanks for identifying the place where the page overflows.
> 
> Rather than changing the styles here, I would shorten the button text.

Even better, I realized the text was the reason for the bug but I didn't want to edit it.

**Comment by AndroGenius-codes** [comment]
> Thanks for the contribution @AndroGenius-codes!

Happy to help

**Pull request merged**